client/trino-jdbc/pom.xml

```diff
@@ -421,6 +421,17 @@
                         <artifactId>surefire-testng</artifactId>
                         <version>${dep.plugin.surefire.version}</version>
                     </dependency>
+                    <!-- temporary workaround for https://github.com/airlift/airbase/pull/670 -->
```

Comment:
How about we wait until that PR lands to update airbase ?

Reply:
Whatever comes first